apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/eobReadCarrierWithMultipleLines.json

```diff
@@ -0,0 +1,2558 @@
+        "code" : "priorpayerpaid",
+        "display" : "Prior payer paid"
+      }, {
+        "system" : "https://bluebutton.cms.gov/resources/codesystem/adjudication",
+        "code" : "https://bluebutton.cms.gov/resources/variables/clm_tot_chrg_amt",
+        "display" : "Claim Total Charge Amount"
+      } ]
+    },
+    "amount" : {
+      "value" : 0.0,
+      "currency" : "USD"
+    }
+  } ],
+  "payment" : {
+    "amount" : {
+      "value" : 199.99,
+      "currency" : "USD"
+    }
+  },
+  "benefitBalance" : [ {
+    "category" : {
+      "coding" : [ {
+        "system" : "http://terminology.hl7.org/CodeSystem/ex-benefitcategory",
+        "code" : "1",
+        "display" : "Medical Care"
+      } ]
+    },
+    "financial" : [ {
+      "type" : {
+        "coding" : [ {
+          "system" : "https://bluebutton.cms.gov/resources/codesystem/benefit-balance",
+          "code" : "https://bluebutton.cms.gov/resources/variables/carr_clm_cash_ddctbl_apld_amt",
+          "display" : "Carrier Claim Cash Deductible Applied Amount (sum of all line-level deductible amounts)"
+        } ]
+      },
+      "usedMoney" : {
+        "value" : 777.0,
+        "currency" : "USD"
+      }
+    }, {
+      "type" : {
+        "coding" : [ {
+          "system" : "https://bluebutton.cms.gov/resources/codesystem/benefit-balance",
+          "code" : "https://bluebutton.cms.gov/resources/variables/nch_clm_prvdr_pmt_amt",
+          "display" : "NCH Claim Provider Payment Amount"
+        } ]
+      },
+      "usedMoney" : {
+        "value" : 123.45,
+        "currency" : "USD"
+      }
+    }, {
+      "type" : {
+        "coding" : [ {
+          "system" : "https://bluebutton.cms.gov/resources/codesystem/benefit-balance",
+          "code" : "https://bluebutton.cms.gov/resources/variables/nch_clm_bene_pmt_amt",
+          "display" : "NCH Claim Payment Amount to Beneficiary"
+        } ]
+      },
+      "usedMoney" : {
+        "value" : 888.0,
+        "currency" : "USD"
+      }
+    }, {
+      "type" : {
+        "coding" : [ {
+          "system" : "https://bluebutton.cms.gov/resources/codesystem/benefit-balance",
+          "code" : "https://bluebutton.cms.gov/resources/variables/nch_carr_clm_sbmtd_chrg_amt",
+          "display" : "NCH Carrier Claim Submitted Charge Amount (sum of all line-level submitted charges)"
+        } ]
+      },
+      "usedMoney" : {
+        "value" : 245.04,
+        "currency" : "USD"
+      }
+    }, {
+      "type" : {
+        "coding" : [ {
+          "system" : "https://bluebutton.cms.gov/resources/codesystem/benefit-balance",
+          "code" : "https://bluebutton.cms.gov/resources/variables/nch_carr_clm_alowd_amt",
+          "display" : "NCH Carrier Claim Allowed Charge Amount (sum of all line-level allowed charges)"
+        } ]
+      },
+      "usedMoney" : {
+        "value" : 166.23,
+        "currency" : "USD"
+      }
+    } ]
+  } ]
+}
```

Comment:
unfortunately since we didn't have a before version of this, we cannot see the difference to assess backwards compatibility. can you provide the diffs of the responses (either as a series of commits) or as a comment?